playground/nuxt.config.ts

```diff
@@ -1,5 +1,5 @@
 import { defineNuxtConfig } from 'nuxt'
-import nuxtMetaModule from '../src/module'
+import nuxtMetaModule from 'nuxt-component-meta'
```

Comment:
By using the package, the playground will not use current changes in development mode and it will use the latest release.

As I checked, your fix works perfectly fine without adding dependencies to the playground and changing `dev:prepare` script